index.html

```diff
@@ -267,17 +272,23 @@ <h2 class="heading">
             </ul>
         </div>
     </section>
+    <!--Contact form-->
     <section class="contact">
         <div class="contact-heading">
-            <div>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</div>
+            <div>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+            </div>
         </div>
         <div class="form">
-            <input class="form-fill" type="text" name="name" placeholder="Full name">
-            <input class="form-fill" type="email" name="email" placeholder="Email address">
-            <textarea name="mail-content" id="" cols="30" rows="5" placeholder="Enter text here"></textarea>
-            <input class="submit" type="submit" value="Get in touch">
+            <form action="https://formspree.io/f/xknarzjq" method="post">
+                <input class="form-fill" type="text" name="name"  pattern="^[a-zA-Z]{3,30}$" placeholder="Full name" required>
```

Comment:
- [ ] You have done really well implementing the contact form, please use the max length attribute to keep make sure Name field accepts up to 30 chars. currently I can enter more than 30 characters, the validation by using the pattern attribute is giving an ambiguous error which might be confusing for the user, I say prefer the maxlength attribute as that is made for that specific purpose :+1: 
![image](https://user-images.githubusercontent.com/97521557/220319689-c93d939d-3082-4b52-93fe-e5928c3a1ba2.png)